frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -107,7 +107,7 @@ const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
           // eslint-disable-next-line react/no-multi-comp
           expandIcon: ({ expanded, onExpand, record }) =>
             record.instances.length ? (
-              <RightOutlined
+              <CaretRightOutlined
```

Comment:
```suggestion
              <CaretRightOutlined className="transition-transform"
```

Reply:
Sorry i forgot to answer you. I haven't added that class because it doesn't seem to work

Reply:
We could have a broken Tailwind install then I guess...
I think we could manually add [this code](https://tailwindcss.com/docs/transition-property) to our codebase.